src/subscription_manager/managercli.py

```diff
@@ -2265,6 +2267,31 @@ def _validate_options(self):
             system_exit(os.EX_USAGE, _("Error: --no-overlap is only applicable with --available"))
         if self.options.pid_only and self.options.installed:
             system_exit(os.EX_USAGE, _("Error: --pool-only is only applicable with --available and/or --consumed"))
+        if self.options.after and not self.options.available:
+            system_exit(os.EX_USAGE, _("Error: --after is only applicable with --available"))
+        if self.options.after and not self.options.all:
+            system_exit(os.EX_USAGE, _("Error: --after is only applicable with --all"))
```

Comment:
this pair of conditions does not make sense. if it passes the first one, then if fails the second and vice versa

shouldn't it be compunded to one condition: 'if self.options.after and not (self.options.available or  self.options.all)'